### ISSUE TITLE
Field' image numbers' doesn't have a associated label element

### ISSUE THREAD

**Issue by hannolans** [UniversalViewer/universalviewer#456]
Field' image numbers' doesn't seem to have a associated label element. For accessibility all form elements needs to have a associated label. (https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/minimize-error-cues.html)

**Comment by edsilv** [comment]
Stanford have done an accessibility audit and added aria-labels to everything on the v3 branch. Would you mind checking if it's ok now?

**Comment by hannolans** [comment]
This seems ok now.
